mandatory/1-syntax-errors.js

```diff
@@ -1,18 +1,22 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
   return a + b + c;
 }
 
-function introduceMe(name, age)
-  return "Hello, my name is " + name "and I am " age + "years old";
+function introduceMe(name, age) {
+  return "Hello, my name is " + name + " and I am " + age + " years old";
+}
+let greet = introduceMe("Sonjide", 27);
+console.log(greet);
 
 function getTotal(a, b) {
-  total = a ++ b;
+  return "The total is " + (a + b);
 
-  return "The total is total";
+  
 }
-
+var result = getTotal(14, 14);
```

Comment:
'var' is now considered outdated. Please use 'let' instead

Reply:
Yeah, I have corrected it.